code_samples/recent_activity/src/Command/MonitorRecentContentCreationCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use Ibexa\Contracts\ActivityLog\ActivityLogServiceInterface;
+use Ibexa\Contracts\ActivityLog\Values\ActivityLog\Criterion;
+use Ibexa\Contracts\ActivityLog\Values\ActivityLog\Query;
+use Ibexa\Contracts\ActivityLog\Values\ActivityLog\SortClause\LoggedAtSortClause;
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class MonitorRecentContentCreationCommand extends Command
+{
+    protected static $defaultName = 'app:monitor-content-creation';
+
+    private ActivityLogServiceInterface $activityLogService;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(ActivityLogServiceInterface $activityLogService, PermissionResolver $permissionResolver, UserService $userService)
+    {
+        parent::__construct(self::$defaultName);
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->activityLogService = $activityLogService;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('List last 10 log entry groups with creations in the last hour')
```

Comment:
In the same manner here, description should be moved to a static property.

Reply:
Done in 4e39143c1d50a46081ef869f238271373875ae63